.github/actions/spell-check/expect.txt

```diff
@@ -1861,6 +1862,7 @@ Winfried
 wireformat
 wirelength
 Wisiol
+wmem
 Wmissing
```

Comment:
Fwiw, from https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns, one could probably adapt some subset of this:
```
# Compiler flags
[\t "'`=]-[DPWXYLlf]
```

To drop `Wmissing` (not sure it's worth it).